src/release.js

```diff
@@ -129,7 +129,7 @@ module.exports = async function ({ github, context, inputs }) {
       try {
         // post a comment about release on npm to any linked issues in the
         // any of the PRs in this release
-        await notifyIssues(github, owner, repo, release)
+        await notifyIssues(github, npmToken, owner, repo, release)
```

Comment:
instead of passing around the npmToken, which is a secret, why don't we introduce a simple boolean which tells us if we need to publish to npm and pass that around instead? They're pretty much the same, but passing around something that's a secret piece of information just to detect if you need to do something or not doesn't sound good

Reply:
The requested change is done for the `notifyIssues`, however the actual secret still needs to be passed on to the `publishToNpm` function.

Reply:
yes that's expected